_posts/0000-01-29-sridhar353.md

```diff
@@ -0,0 +1,2 @@
+This is Line 1
+This is line 2
```

Comment:
Image is missing

Reply:
That's right. I added a text file instead. Thanks for the comment